openhands/server/conversation_manager/standalone_conversation_manager.py

```diff
@@ -66,22 +66,29 @@ async def attach_to_conversation(self, sid: str) -> Conversation | None:
             if sid in self._active_conversations:
                 conversation, count = self._active_conversations[sid]
                 self._active_conversations[sid] = (conversation, count + 1)
-                logger.info(f'Reusing active conversation {sid}')
+                logger.info(
+                    f'Reusing active conversation {sid}', extra={'session_id': sid}
```

Comment:
```suggestion
                    f'Reusing active conversation , extra={'session_id': sid}
```

We can maybe remove the duplicate `sid`, since `extra` is default?

Reply:
That wasn't ready to happen, because these extra fields were only logging in json mode. But it seems desirable and consistent to log them in plain text mode as well, so I've now added ExtraFieldsFilter to do that, enabling sid to be removed from these message strings.

Let me know what you think, we could rollback those commits if needed.

Reply:
Oooh, I'm really sorry! I think it was a pretty bad idea from my part. So apart from what the monitoring stuff that I'm not referring to - you know what you want there.

There are two aspects I'd take into account, and none is blocking or anything, but it may be useful to think about:
The first is that we have these extra things in the code, so when we introduce yet another approach it would be nice to do something about them; reuse them, absorb them, remove them - hopefully the way we use things can be consistent. (maybe, maybe it can't)

The second is reality lol. People like logging with f-strings. They do it all the time. In the past we had logging with % lazy formatting, and no one used it 😅 - and at some point I must have removed the last traces of it.

If we try to get folks to replace f-strings with an extra dict... we may be doomed to fail... 😓 🤷

What do you think?

Reply:
Gotcha. For now I've rolled that addition back.

We don't necessarily need every log message to include every piece of metadata it can, so if people don't change their behavior that's fine. The adapters can help it happen by default within scopes where it makes sense, leaving a minimal number of times we have an `extra=`.

The `log()` method on AgentController is another approach, I lean toward replacing it with the adapter since it's doing the same thing in a way that doesn't match the standard logger api. It would also be possible to use that pattern across the board instead of adapters - i.e. Agent and AgentSession get a similar `log` method.